README.md

```diff
@@ -214,6 +215,12 @@ Please note that:
 - Please bear in mind that the Grants Committee might be stricter in accepting maintainers when compared to typical grants, mostly selecting for applicants with proven experience in the relevant tech stacks.
 - Maintenance Grants are only awarded for fixed timeframes. The requested duration needs to be specified in the application.
 
+## :moneybag: Referral Program
+
+We give away 1000 USD to each referral of a successful grant application by _anyone having previously worked on a Web3 Foundation grant_ or _a [Polkadot Ambassador](https://wiki.polkadot.network/docs/ambassadors)_. Web3 Foundation and Parity employees do not qualify for the program, even if they previously worked on a grant.
```

Comment:
It's not clear to me if this is restricted to grantees who completed their grant, or at least 1 milestone or if no restrictions at all apply. Maybe we could set the barrier to grantees who delivered at least 1 milestone?

Reply:
I think this would make it more complicated to check it and is not necessary. As soon as someone's grant application is merged, he can refer someone else.